js/gdm/authPrompt.js

```diff
@@ -15,6 +15,7 @@ const Animation = imports.ui.animation;
 const Batch = imports.gdm.batch;
 const Config = imports.misc.config;
 const GdmUtil = imports.gdm.util;
+const InputSourceManager = imports.ui.status.keyboard;
```

Comment:
Is there any reason why this import is `InputSourceManager` instead of `Keyboad` (in reference to the name of the module imported)?